test/units/testsuite-58.sh

```diff
@@ -773,6 +773,58 @@ EOF
     systemd-dissect -U "$imgs/mnt"
 }
 
+test_minimize() {
+    local defs imgs output
+
+    if systemd-detect-virt --quiet --container; then
+        echo "Skipping verity test in container."
+        return
+    fi
+
+    defs="$(mktemp --directory "/tmp/test-repart.XXXXXXXXXX")"
+    imgs="$(mktemp --directory "/var/tmp/test-repart.XXXXXXXXXX")"
+    # shellcheck disable=SC2064
+    trap "rm -rf '$defs' '$imgs'" RETURN
+
+    for format in ext4 btrfs vfat; do
+        if command -v "mkfs.$format" >/dev/null; then
```

Comment:
Shouldn't this condition be negated?

Reply:
God I need to run these tests again when making changes, you're right ofc